x-pack/plugins/cases/public/components/user_actions/property_actions/alert_property_actions.test.tsx

```diff
@@ -17,84 +17,87 @@ import {
 } from '../../../common/mock';
 import { AlertPropertyActions } from './alert_property_actions';
 
-// FLAKY: https://github.com/elastic/kibana/issues/174667
-describe.skip('AlertPropertyActions', () => {
-  let appMock: AppMockRenderer;
-
-  const props = {
-    isLoading: false,
-    totalAlerts: 1,
-    onDelete: jest.fn(),
-  };
-
-  beforeEach(() => {
-    jest.clearAllMocks();
-    appMock = createAppMockRenderer();
-  });
+for (let i = 0; i < 200; i++) {
+  describe('AlertPropertyActions', () => {
+    let appMock: AppMockRenderer;
+
+    const props = {
+      isLoading: false,
+      totalAlerts: 1,
+      onDelete: jest.fn(),
+    };
+
+    beforeEach(() => {
+      jest.clearAllMocks();
+      appMock = createAppMockRenderer();
+    });
 
-  it('renders the correct number of actions', async () => {
-    appMock.render(<AlertPropertyActions {...props} />);
+    it('renders correctly', async () => {
+      appMock.render(<AlertPropertyActions {...props} />);
```

Comment:
What is going on here!? 😆 

Maybe something gets cached between these and when `'renders the correct number of actions'` has to call `appMock.render` again it gets faster??

Reply:
Maybe it's because of memoization of propertyActions on line 29 in `x-pack/plugins/cases/public/components/user_actions/property_actions/alert_property_actions.tsx`